desci-server/src/utils/discordUtils.ts

```diff
@@ -2,11 +2,64 @@ import { DiscordNotification } from '@penseapp/discord-notification';
 
 import { logger } from '../logger.js';
 
-const discordNotification = new DiscordNotification(
+const doiMintingNotification = new DiscordNotification(
+  process.env.SERVER_URL,
+  process.env.DISCORD_NOTIFICATIONS_DOI_WEBHOOK_URL,
+);
+
+const nodeFeedNotification = new DiscordNotification(
   process.env.SERVER_URL,
   process.env.DISCORD_NOTIFICATIONS_WEBHOOK_URL,
 );
-export const discordNotify = async (message: string) => {
+
+export enum DiscordNotifyType {
+  SUCCESS,
+  INFO,
+  WARNING,
+  ERROR,
+}
+
+export enum DiscordChannel {
+  NodesFeed,
+  DoiMinting,
+}
+
+type Message =
+  | DiscordNotification['sucessfulMessage']
+  | DiscordNotification['infoMessage']
+  | DiscordNotification['warningMessage']
+  | DiscordNotification['errorMessage'];
+
+export const discordNotify = async ({
+  message,
+  title = 'Node Updated',
+  type = DiscordNotifyType.SUCCESS,
+  channel = DiscordChannel.NodesFeed,
+}: {
```

Comment:
I'd like to see this taking an `env` param as well that we bake into the discord message. Make it a bit more obvious what the source of the event is :eyes: